### PR TITLE
docs: Add note about giving credit to ported tools

### PR DESCRIPTION
**Pull request by fidgetingbits**
As discussed on discord, suggest people give proper credit to ported tools. This includes some minor auto linting from my editor, which I can remove if it matters.

### CODE REVIEW

**Comment by gsingh93** [comment]
+1 for being mindful of the license.

Regarding credit, I don't like the practice of doing it in source code. Maybe we have a separate file for this type of credit, could even be on the website for more visibility.

**Comment by fidgetingbits** [comment]
Ya a separate file makes sense, especially since many tools will be broken across many files in the end so hard to say what one to put the comment in. Any ideas on what the file should be? CREDITS.md?

**Comment by fidgetingbits** [comment]
Updated to suggest using `CREDITS.md` now.

File: DEVELOPING.md [hunk 1]
```diff
@@ -123,7 +124,14 @@ TODO: There are many places GDB shows docstrings, and they show up slightly diff
...
 ## Triggers
 
-TODO: If we want to do something when user changes config/theme - we can do it defining a function and decorating it with `pwndbg.config.Trigger`.
+TODO: If we want to do something when user changes config/theme - we can do it defining a function and decorating it
+with `pwndbg.config.Trigger`.
+
+# Porting public tools
+
+If porting a public tool to pwndbg, please make a point of crediting the original author in the pwndbg source code. This
+can added to [CREDITS.md](./CREDITS.md) noting the original author/inspiration, and linking to the original tool/article. Also
```
Comment (disconnect3d) [inline DEVELOPING.md]:
```suggestion
can be added to [CREDITS.md](./CREDITS.md) noting the original author/inspiration, and linking to the original tool/article. Also
```

**Comment by disconnect3d** [comment]
CREDITS.md works for me. We can change it if anyone has better suggestion

File: DEVELOPING.md [hunk 2]
```diff
@@ -123,7 +124,14 @@ TODO: There are many places GDB shows docstrings, and they show up slightly diff
 
 ## Triggers
 
-TODO: If we want to do something when user changes config/theme - we can do it defining a function and decorating it with `pwndbg.config.Trigger`.
+TODO: If we want to do something when user changes config/theme - we can do it defining a function and decorating it
+with `pwndbg.config.Trigger`.
+
+# Porting public tools
+
+If porting a public tool to pwndbg, please make a point of crediting the original author in the pwndbg source code. This
```
Comment (gsingh93) [inline DEVELOPING.md:132]: I still think "in the pwndbg source code" should be removed here.

Comment (fidgetingbits) [reply DEVELOPING.md:132]: https://github.com/pwndbg/pwndbg/pull/2169 new change here since this already merged

**Pull request merged**